suite-common/wallet-utils/src/accountUtils.ts

```diff
@@ -1177,3 +1178,15 @@ export const isTokenMatchesSearch = (token: TokenInfo, search: string) => {
         token.policyId?.toLowerCase().includes(search)
     );
 };
+
+export const isNftMatchesSearch = (token: Token, search: string) => {
+    return (
+        token.symbol?.toLowerCase().includes(search) ||
```

Comment:
TODO note for myself: our symbol is kinda handicaped by shortening in blockchain-link, we should check that, it breaks searching because there are `...`